scopes/compilation/bundler/bundler.main.runtime.ts

```diff
@@ -12,8 +12,13 @@ import { BundlerContext } from './bundler-context';
 import { devServerSchema } from './dev-server.graphql';
 import { DevServerService } from './dev-server.service';
 import { BundlerService } from './bundler.service';
+import { DevServer } from './dev-server';
+
+type WebpackDevServer = DevServer & { config: { plugins: any[] } };
+export type BundlerConfigSlotFn = (devServer: WebpackDevServer, { envId }: { envId: string }) => WebpackDevServer;
```

Comment:
since this get the entire dev server and return dev server it shouldn't contain the word config. as theoretically the registrar can do something else with the dev server which is not about it's config but something else. like, change its id maybe. or add new prop to it.
Also, right now it's not about bundler as a whole, but specifically for dev servers.
I would call it:
`DevServerTransformer`